onnxconverter_common/onnx_fx.py

```diff
@@ -263,6 +263,15 @@ def load(path, name=None, inputs=None, outputs=None):
         g._bind(oxml, inputs=inputs, outputs=outputs)
         return g
 
+    @staticmethod
+    def load_oxml(oxml, name=None, inputs=None, outputs=None):
```

Comment:
can we merge this function with the existing load, like (path_or_model, ...)?

Reply:
done